deeplake/util/spinner.py

```diff
@@ -29,7 +29,7 @@ def __getattr__(self, attr):
 @contextlib.contextmanager
 def run_spinner(spinner):
     try:
-        if not isinstance(sys.stdout, DummyFile):
+        if not isinstance(sys.stdout, DummyFile) and "pytest" not in sys.modules:
```

Comment:
let's instead create a constant to disable spinner that will be changed when tests are run